tuf: use bundled trusted root if available (#542)
sigstore/sigstore-python: CHANGELOG.md

  ([#535](https://github.com/sigstore/sigstore-python/pull/535))
* Revamped error diagnostics reporting. All errors with diagnostics now implement
  `sigstore.errors.Error`.

> * TUF now fetches its trust materials from a single target, if it is available.
>   ([#542](https://github.com/sigstore/sigstore-python/pull/542))

Reviewer: ```suggestion
* Trust root materials are now retrieved from a single trust bundle,
  if it is available via TUF
  ([#542](https://github.com/sigstore/sigstore-python/pull/542))
```
